src/scenes/_Router/GalleryRoute.tsx

```diff
@@ -70,8 +70,8 @@ export default function GalleryRoute({
       return <GlobalNavbar queryRef={query} />;
     }
 
-    return <Filler tallVariant={isMobile} />;
-  }, [navbar, isMobile, query]);
+    return <Filler tallVariant={isMobile && footer} />;
```

Comment:
it looked like `tallVariant` was used to account for when the footer was taller, so I don't think it needs to be applied if there is no footer.

It was making the filler unnecessarily taller when there was no footer.
@Robinnnnn lmk if it might affect anything else that I might've missed!

Reply:
this should be fine- i wanna refactor all of this anyway! thanks kaito